### ISSUE TITLE
[2pt] UI: schema validation errors are not shown

### ISSUE THREAD

**Issue by HustonMmmavr** [tarantool/cartridge#1082]
Steps to reproduce:

1. Insert this schema `/admin/cluster/schema`

```yml
spaces:
  test_space:
    # engine must be memtx or vinyl
    engine: engine 
    is_local: false
    temporary: false
    format:
    - name: x
      type: unsigned
      is_nullable: false
    indexes:
    - name: pk
      type: TREE
      unique: true
      parts:
      - path: x
        is_nullable: false
        type: unsigned
```
2. Run validate. As a result there is no error shown, but response contains it

![Screenshot from 2020-10-05 12-48-34](https://user-images.githubusercontent.com/25960394/95064987-1f876880-0709-11eb-9a65-a4e216f70467.png)

**Comment by vasiliy-t** [comment]
To see validation error one should scroll down to the error message, we have project with about 2k lines in schema, it'a long scroll.

<img width="1428" alt="Screenshot 2020-10-05 at 13 55 21" src="https://user-images.githubusercontent.com/4940266/95071564-878e7c80-0712-11eb-9b82-fe63b668ec3c.png">

**Comment by rosik** [comment]
Resolved in #1492
